src/screens/Grommet.js

```diff
@@ -343,6 +343,22 @@ const GrommetPage = () => (
       multiple: "string",
       selected: "string",
     },
+    selectMultiple: {
+      clearAll: "string";
```

Comment:
```suggestion
      clearAll: "string",
```
Looks good just need to swap `;` to `,`

Reply:
aw yes